src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CachingCodeFixProviderForProjects.cs

```diff
@@ -84,12 +85,15 @@ private ImmutableArray<CodeFixProvider> LoadFrom(Project project)
                         return null;
                     }
                 })
-                .Where(x => x != null)
-                .ToImmutableArray();
+                .Where(x => x != null);
+
+            var builtInCodeFixes = _providers.SelectMany(provider => provider.CodeFixProviders);
```

Comment:
this is another example of something that could be handled in a separate PR 🙈

Reply:
Yeah i got bit sidetracked 😄

Reply:
Yeah got bit sidetracked 😄